CredentialProvider.Microsoft/Util/EnvUtil.cs

```diff
@@ -21,6 +21,7 @@ public static class EnvUtil
         public const string AdalFileCacheEnvVar = "NUGET_CREDENTIALPROVIDER_ADAL_FILECACHE_ENABLED";
         public const string PpeHostsEnvVar = "NUGET_CREDENTIALPROVIDER_ADAL_PPEHOSTS";
 
+        public const string DeviceFlowTimeoutEnvVar = "NUGET_CREDENTIALPROVIDER_VSTS_DEVICEFLOWTIMEOUTINSECONDS";
```

Comment:
nit: For consistency with 'NUGET_CREDENTIALPROVIDER_VSTS_SESSIONTIMEMINUTES', drop the 'IN'